tools/extra/pac/fpgabist/bist_common.py

```diff
@@ -130,7 +131,7 @@ def load_gbs(gbs_file, bdf):
     print "Attempting Partial Reconfiguration:"
     cmd = ['fpgaconf', '-B', hex(bdf['bus']), '-D',
            hex(bdf['device']), '-F', hex(bdf['function']),
-           gbs_file]
+           '-v', gbs_file]
```

Comment:
I think that passing fpgaconf the -v option will invoke the help message and exit.  If verbose output is required, then use -V.  This change was made last fall on master.